pydantic_ai_slim/pydantic_ai/realtime/openai.py

```
@@ -0,0 +1,757 @@
+        return events
+
+    async def _try_reconnect(self) -> bool:
+        """Re-dial with exponential backoff; return whether a new connection was established."""
+        assert self._reconnect is not None and self._dial is not None
+        return await reconnect_with_backoff(self._reconnect, self._attempt_reconnect)
+
+    async def _attempt_reconnect(self) -> bool:
+        assert self._dial is not None
+        try:
+            self._ws = await self._dial()
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)